__init__.py

```diff
@@ -273,40 +276,41 @@ def shutdown(self):
         self.bus.remove('recognizer_loop:audio_output_end',
                         self.on_handler_mouth_reset)
 
-    def on_handler_started(self, message):
-        handler = message.data.get("handler", "")
-        # Ignoring handlers from this skill and from the background clock
-        if 'Mark2' in handler:
-            return
-        if 'TimeSkill.update_display' in handler:
-            return
-
     def on_handler_mouth_reset(self, message):
-        """ Restore viseme to a smile. """
+        """Restore viseme to a smile."""
         pass
 
+    def on_handler_started(self, message):
+        """When a skill begins executing turn on the LED ring"""
+        handler = message.data.get('handler', '')
+        if _skip_handler():
```

Comment:
Should be `self._skip_handler()` maybe and there should probably be an argument as well